appcenter-auth/android/build.gradle

```
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion rootProject.properties.get('compileSdkVersion', 26)
+    buildToolsVersion rootProject.properties.get('buildToolsVersion', '27.0.3')
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion rootProject.properties.get('targetSdkVersion', 26)
+        versionCode 41
+        versionName '2.0.0'
+        ndk {
+            abiFilters 'armeabi-v7a', 'x86'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    //noinspection GradleDynamicVersion
+    implementation 'com.facebook.react:react-native:+'
+    api 'com.microsoft.appcenter:appcenter-auth:2.1.0-7+df4dfd7f7'
```

```suggestion
    api 'com.microsoft.appcenter:appcenter-auth:2.1.0-22+12baf443e'
```
So that you have the new APIs we added (accessToken and idToken in addition to accountId), and UserInformation in the right package.

---

Changed